cloud/aws/benchmarks/benchmarks.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# The caller should download the static-tests and extract them into the
+# local directory.
+
+set -x
+
+TEST_FLAGS="-test.bench=.* -test.benchmem -test.run=NONE -test.count=10"
```

Comment:
`go test -bench` runs the benchmarks for a fixed amount of time, so that's to be expected. The question is whether the number of iterations reported in the benchmark results differ.